docs/reference/changelog.rst

```diff
@@ -31,6 +31,12 @@ New features:
   pseudo-partition. To make use of this feature, the project must support both overlays
   and partitions.
 
+- Add a new LifecycleManager creation parameter to control whether parts should have a
+  "usrmerged" directory structure pre-created in their install directories.
+
+- Add two new build-attributes, "enable-usrmerge" and "disable-usrmerge", to control the
+  creation of the "usrmerged" directory structure in a part's install dir.
```

Comment:
I feel like in some quarters this will be seen as a _big_ change, but I'm not really sure how best to promote it in Craft Parts itself. It will certainly earn a major release note in our apps.

Reply:
Definitely. The plan is this: Rockcraft will be the first app to enable the new default behavior, only for rocks targeting base `ubuntu@25.10` and later. This gives us some time while the base is still unstable to experiment with the feature, and then once the 26.04 LTS rolls around this should be solid and the impact reasonable well-known.

This will feature prominently in Rockcraft's release notes regarding 25.10 support.